packages/nextra-theme-blog/src/index.js

```diff
@@ -13,13 +13,17 @@ import getTitle from './utils/get-title'
 import getTags from './utils/get-tags'
 import sortDate from './utils/sort-date'
 
+// comments
+import { ReactCusdis } from 'react-cusdis'
```

Comment:
```suggestion
import dynamic from 'next/dynamic'

const ReactCusdis = dynamic(
  () => import('react-cusdis').then((mod) => mod.ReactCusdis),
  { ssr: false }
)
```

With dynamic import, the module won't be included in the bundle if the `cusdis` option is not enabled.